go/ct/gen/stack.go

```diff
@@ -94,9 +95,8 @@ func (g *StackGenerator) BindValue(pos int, variable Variable) {
 }
 
 func (g *StackGenerator) Generate(assignment Assignment, rnd *rand.Rand) (*st.Stack, error) {
-	// convert variable constraints to constant constraints
-	constraints := make([]constValueConstraint, len(g.constValues), len(g.constValues)+len(g.varValues))
-	copy(constraints, g.constValues)
+	constraints := utils.RightPadSliceWithCap(g.constValues, int64(len(g.constValues)), int64(len(g.constValues)+len(g.varValues)))
+
```

Comment:
This is a special case that semantically is not even a padding. I think this should remain as it used to be.